roman_numeral/roman_numerals.py

```diff
@@ -0,0 +1,52 @@
+"""Given a roman numeral, find the integer equivalent"""
+
+# roman_numerals = ["I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IV", "X"]
+
+
+number = 15
+
+roman_numeral_values = {1: "I", 5: "V", 10: "X"}
+
+
+def conversion(number):
+    """Convert an Arabic number to its roman numeral equivalent"""
+
+    if number == 0:
+        return ""
+
+    else:
+        convert = []
+        reduced_num = number
+
+        # First check: see if number is in dictionary. If so, return it, if not get the largest value from the dictionary
+        if number % 5 == 0:
+            try:
```

Comment:
Look closely at lines 23-28 and see if there's duplication that you can eliminate...same with lines 34-41.

Reply:
hint, you probably don't need to use the `try-except` here.